packages/driver/cypress/fixtures/nested/3975_a.html

```diff
@@ -1,4 +1,14 @@
 <html>
+  <head>
+    <meta charset="utf-8" />
+    <!--
+      Our initial attempts to fix issue 3785 caused issues on pages Disallowing 'unsafe-eval'.
+      Including this meta tag ensures that #3785 remains fixed without reintroducing #19697.
+
+      https://github.com/cypress-io/cypress/issues/19697
+    !-->
+    <meta http-equiv="Content-Security-Policy" content="script-src 'unsafe-inline'" />
```

Comment:
The repro shared in #19697 uses `<meta http-equiv="Content-Security-Policy" content="script-src 'self'" />`, but that didn't actually recreate the issue when we tried it. What's with the change to `script-src 'unsafe-inline'`? Was this the actual repro for #19697 or something?

Reply:
Self means "only allow script tags loaded from this domain". We need `unsafe-inline' because `self` throws an error for the inline <script>` tag in this fixture.

I've still been unable to get their test-tiny example to fail - my suspicion is that it has to do with the http headers their server is sending. I did verify that this test fails without the code change and passes with it in-place.

Now for the extra confusing bit. @pmattj reports "Incidentally, I'm running 3975_spec.js locally with your updated 3975_a.html on a fresh build of 9.3.1, and it seems to pass without the other changes in your PR." - which doesn't match what's going on on my machine.